hack/gen-resolved-issue.py

```diff
@@ -1039,17 +1058,27 @@ def triage_regressions(regressed_tests, triaged_incidents, issue_url, currrent_c
     file_matches = None
     if args.file_matches:
         try:
-            # file_matches = json.load(args.file_matches)
             file_matches = args.file_matches
         except Exception as e:
-            print(f"Failed to load json file matches: {e}")
+            print(f"Failed to load file matches: {e}")
             exit()
 
+    test_details = None
+    if args.test_details:
+        try:
+            test_details = args.test_details
+        except Exception as e:
+            print(f"Failed to load test details: {e}")
+            exit()        
```

Comment:
i'm confused about this whole pattern. since you're not doing any json/file load here, i don't think you need to check for Exceptions unless there's something weird going on with attr access. could just be:
```
file_matches = args.file_matches or None
test_details = args.test_details or None
```
... or am i reading this wrong?